cmd/cerbos/compile/internal/compilation/display.go

```diff
@@ -18,6 +20,8 @@ func Display(p *printer.Printer, errs compile.ErrorList, output flagset.OutputFo
 		return displayJSON(p, errs, colorLevel)
 	case flagset.OutputFormatList, flagset.OutputFormatTree:
 		return displayList(p, errs)
+	case flagset.OutputFormatJUnit:
```

Comment:
When there's a compilation failure, it won't get output to the screen because of this. Try running `compile` on a directory that contains an invalid policy. 

```
cerbos: error: invalid option junit
```

We need to either introduce a new flag for test results output format or default to outputting compile errors in one of the supported formats.